backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('/users')
+export class UserController {
+constructor(private readonly userService: UserService) {
+}
+    @Get(':id')
+    getUserById(@Param('id') id: string)
+    {
+        const data = {id: "id"}
```

Comment:
tutaj endpoint ma zaciagać z bazy danych stosownego usera, obecnie otrzymujemy tylko to co wysłaliśmy